semaphoreci.sh

```diff
@@ -75,7 +75,7 @@ for PREREQ in ${GCC_PREREQS}; do
 done
 
 ## Create the build directory.
-# Build typically takes around 10 minutes, could this be cached across CI runs?
+# Build typically takes around 10 minutes with -j4, could this be cached across CI runs?
 mkdir ${SEMAPHORE_PROJECT_DIR}/build
```

Comment:
Yeah, I gave it a try, and it didn't seem like semaphore was keeping the tarballs I was putting there.

Then again, it doesn't seem to be keeping the prerequisites in cache either.  Maybe semaphore doesn't keep cache created by PRs, but then again, that would be reasonable behaviour.